musicbrainzngs/mbxml.py

```diff
@@ -523,6 +525,9 @@ def parse_cdstub(cdstub):
 
     return result
 
+def parse_offset_list(ol):
```

Comment:
The only test for `get_releases_by_discid` tests the created WS url, not the result. (`test_getentity`).

I'll see if I can create a `test_mbxml_disc.py` or similar.